commands/app.go

```diff
@@ -29,7 +29,7 @@ func Run(args []string) {
 			Name:      "login",
 			Usage:     "Log in to LeanCloud",
 			Action:    wrapAction(loginAction),
-			ArgsUsage: "[-u username -p password (--region <CN> | <US> | <TAB>)]",
+			ArgsUsage: "[-u username -p password (--region <cn-n1> | <us-w1> | <cn-e1>)]",
```

Comment:
```suggestion
			ArgsUsage: "[-u username -p password (--region <cn-n1> | <cn-e1> | <us-w1>)]",
```

把顺序调了下，由国内到国外。（我猜之前那样排列仅仅是因为华东是后加的。